styles.css

```diff
@@ -0,0 +1,78 @@
+html,
+body {
+  height: 100%;
+  font-family: 'Open Sans', sans-serif;
+  text-align: center;
+}
+
```

Comment:
the CSS style we're doing (which is an agreed-upon Interwebs standard) doesn't have newlines in between definitions